bin/load-spark-env.sh

```diff
@@ -47,8 +47,8 @@ if [ -z "$SPARK_SCALA_VERSION" ]; then
   ASSEMBLY_DIR1="${SPARK_HOME}/assembly/target/scala-2.12"
 
   if [[ -d "$ASSEMBLY_DIR2" && -d "$ASSEMBLY_DIR1" ]]; then
-    echo -e "Presence of build for multiple Scala versions detected." 1>&2
-    echo -e 'Either clean one of them or, export SPARK_SCALA_VERSION in spark-env.sh.' 1>&2
+    echo -e "Presence of build for both scala versions(SCALA 2.11 and SCALA 2.12) detected." 1>&2
```

Comment:
How about something like "Multiple Scala versions detected ($ASSEMBLY_DIR1 and $ASSEMBLY_DIR2). Remove one, or export SPARK_SCALA_VERSION=(version) in load-spark-env.sh"

That adds more info and maybe is less maintenance as we add or drop Scala versions.

Reply:
Make sense. Let me have a quick update